DeathNotes.cs

```diff
@@ -16,7 +16,7 @@ namespace Oxide.Plugins
     using WeaponPrefabs = DeathNotes.RemoteConfiguration<Dictionary<string, string>>;
     using CombatEntityTypes = DeathNotes.RemoteConfiguration<Dictionary<string, DeathNotes.CombatEntityType>>;
 
-    [Info("Death Notes", "LaserHydra", "6.3.5")]
+    [Info("Death Notes", "LaserHydra", "6.3.7")]

```

Comment:
Please don't increment the version yet. I will do that once before merging develop into master.
(I just created the develop branch now)